check/format-incremental

```diff
@@ -109,6 +105,11 @@ if (( any_files == 0 )); then
     exit 0
 fi
 
+# TODO: Remove '-e cirq/ops/pauli_string.py' once the line is covered (https://github.com/quantumlib/Cirq/issues/3804)
+flynt ${format_files} --quiet -f -e cirq/ops/pauli_string.py
+FLYNTSTATUS=$?
+
+echo "Running black..."
```

Comment:
people may not know this is a formatter. sounds kinda funny

Reply:
oh, that is so true :D